fpdf/fpdf.py

```diff
@@ -2235,6 +2235,42 @@ def text_annotation(
         self.pages[self.page].annots.append(annotation)
         return annotation
 
+    @check_page
+    def free_text_annotation(
+        self,
+        x,
+        y,
+        text,
+        w=1,
+        h=1,
+        flags=DEFAULT_ANNOT_FLAGS,
+    ):
+        """
+        Puts a free text annotation on a rectangular area of the page.
+
+        Args:
+            x (float): horizontal position (from the left) to the left side of the link rectangle
+            y (float): vertical position (from the top) to the bottom side of the link rectangle
+            text (str): text to display
+            w (float): optional width of the link rectangle
+            h (float): optional height of the link rectangle
```

Comment:
Just a suggestion on the function [signature][https://en.wikipedia.org/wiki/Type_signature):
some parameters could be made optionals:
* `x` & `y`,  if not provided, could be set from `self.x` & `self.y`
* `h`, if not provided, could be set to the current font size, as already done in `FPDF.cell()`, `FPDF.multi_cell()` & `FPDF.ln()`
* `w`, if not provided, could be set from `FPDF.get_string_width()`, as already done in `FPDF.text()` & `FPDF.cell()`

In fact, only the `text` parameter is really mandatory I think. For that reason, it should probably be the first paramater of this method.